bindings/bindingtester/tests/api.py

```diff
@@ -570,18 +581,39 @@ def generate(self, args, thread_number):
                 instructions.push_args(key1, key2, chunkSize)
                 instructions.append(op)
                 self.add_strings(1)
-
+            elif op == 'TENANT_CREATE':
+                tenant_name = self.choose_tenant(0.8)
+                self.allocated_tenants.add(tenant_name)
+                instructions.push_args(tenant_name)
+                instructions.append(op)
+                self.add_strings(1)
```

Comment:
What's the purpose of add_strings here?

Reply:
The `add_strings` function is helping to do some accounting about what is on the top of the stack. In particular it is saying that in running this operation, we pushed something onto the stack that can be used as a string, which makes it legal to use as the value of a mutation.